content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -83,24 +78,127 @@ Resource associated with Activity:
     service.instance.id: 20c891c2-94b4-4203-a960-93a22e837a32
 ```
 
+</details>
+
 This output matches the span created in the preceding code sample.
 
+### Metrics
+
+Next, add metrics to the tracing code. This will initialize a
+[Meter](/docs/concepts/signals/metrics), and will use the Meter to create a
+Counter.
+
+```csharp
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+// Define some important constants to initialize tracing with
+var serviceName = "MyCompany.MyProduct.MyService";
+var serviceVersion = "1.0.0";
+
+// Factor out the ResourceBuilder so it can be shared with the tracer provider
+// and the meter provider.
+var appResourceBuilder = 
+    ResourceBuilder.CreateDefault()
+        .AddService(serviceName: serviceName, serviceVersion: serviceVersion);
+
+var MyActivitySource = new ActivitySource(serviceName);
+
+// Configure important OpenTelemetry settings and the console exporter
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddSource(serviceName)
+    .SetResourceBuilder(appResourceBuilder)
+    .AddConsoleExporter()
+    .Build();
+
+// Now configure Metrics
+var meter = new Meter("Example");
+var counter = meter.CreateCounter<long>("Example.Counter");
```

Comment:
a common ask users have is how/where to create meter and instruments. While this example is good as is, if we can show how to do these in controller class, it would be nicer.